### PR TITLE
Fix for embedding tests and others

### PR DESCRIPTION
**Pull request by ogibson**
# Background
#### Link to issue 

https://biomage.atlassian.net/browse/BIOMAGE-777

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ X] Tested locally with Inframock
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ X] Photo of a cute animal attached to this PR
image

### CODE REVIEW

File: python/src/tests/tasks/test_embedding.py [hunk 1]
```diff
@@ -80,4 +79,3 @@ def test_tsne_edits_object_appropriately(self):
     def test_throws_on_invalid_embedding_type(self):
         with pytest.raises(Exception):
             ComputeEmbedding(self).compute("definitelynotavalidembedding")
```
Comment (xverges) [inline python/src/tests/tasks/test_embedding.py:81]: Does not look like we are testing what we say we are testing 🤔 . I think that we need to create something like
```
            "body": {
                "name": "GetEmbedding",
                "type": "definitelynotavalidembedding",
            }
```

Comment (ivababukova) [reply python/src/tests/tasks/test_embedding.py:81]: Actually, I think in this case the title of the test is wrong, but the actual test contents are correct. If you look at this line: https://github.com/biomage-ltd/worker/blob/master/python/src/tasks/embedding.py#L25 there is no validation of which task type we are doing. Validation of task name happens here: https://github.com/biomage-ltd/worker/blob/master/python/src/tasks/tasks.py#L57 So this test basically checks what will happen if there is no `task_def` defined in `self` of the class.

Comment (ivababukova) [reply python/src/tests/tasks/test_embedding.py:81]: so I think just renaming the title of the test to should work

**Review by ivababukova — APPROVED** [review]
let's merge after you fix the test name

**Pull request merged**